packages/malloy/src/dialect/dialect.ts

```diff
@@ -172,6 +177,18 @@ export abstract class Dialect {
   // Like characters are escaped with ESCAPE clause
   likeEscape = true;
 
+  // Added field supportsLimit. TSQL Doesn't support limit and limit is not part of the SQL Standard
+  // TODO (vitor): Discuss with malloy
+  supportsLimit = true;
+
+  // Added field supportsRegexp. TSQL Doesn't support regexp. Idk how to use this with = false yet (tsql case). Malloy seems to often go for regexp
+  // TODO (vitor): Discuss with malloy
+  supportsRegexp = true;
+
+  // TSQL doesn't let you do complex operations on GROUP BY for aliased columns.
```

Comment:
I think you get to name it. If a second dialect comes along and you picked a wrong name, we'll just change it then.